portfolio/src/main/webapp/script.js

```diff
@@ -12,11 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a fun fact to the page.
+/*
+ Adds a fun fact to the page.
  */
```

Comment:
```suggestion
/**
 * Adds a fun fact to the page.
 */
```

These are doc comments that have special meaning